packages/apps/desktop-viewer-test/package.json

```diff
@@ -32,39 +32,39 @@
   ],
   "dependencies": {
     "@bentley/icons-generic-webfont": "^1.0.15",
-    "@itwin/appui-abstract": "^3.0.0",
-    "@itwin/appui-layout-react": "^3.0.0",
-    "@itwin/appui-react": "^3.0.0",
-    "@itwin/components-react": "^3.0.0",
-    "@itwin/core-backend": "^3.0.0",
-    "@itwin/core-bentley": "^3.0.0",
-    "@itwin/core-common": "^3.0.0",
-    "@itwin/core-electron": "^3.0.0",
-    "@itwin/core-frontend": "^3.0.0",
-    "@itwin/core-geometry": "^3.0.0",
-    "@itwin/core-i18n": "^3.0.0",
-    "@itwin/core-markup": "^3.0.0",
-    "@itwin/core-orbitgt": "^3.0.0",
-    "@itwin/core-quantity": "^3.0.0",
-    "@itwin/core-react": "^3.0.0",
-    "@itwin/core-telemetry": "^3.0.0",
-    "@itwin/ecschema-metadata": "^3.0.0",
+    "@itwin/appui-abstract": "^3.1.0-dev.42",
+    "@itwin/appui-layout-react": "^3.1.0-dev.42",
+    "@itwin/appui-react": "^3.1.0-dev.42",
+    "@itwin/components-react": "^3.1.0-dev.42",
+    "@itwin/core-backend": "^3.1.0-dev.42",
+    "@itwin/core-bentley": "^3.1.0-dev.42",
+    "@itwin/core-common": "^3.1.0-dev.42",
+    "@itwin/core-electron": "^3.1.0-dev.42",
+    "@itwin/core-frontend": "^3.1.0-dev.42",
+    "@itwin/core-geometry": "^3.1.0-dev.42",
+    "@itwin/core-i18n": "^3.1.0-dev.42",
+    "@itwin/core-markup": "^3.1.0-dev.42",
+    "@itwin/core-orbitgt": "^3.1.0-dev.42",
+    "@itwin/core-quantity": "^3.1.0-dev.42",
+    "@itwin/core-react": "^3.1.0-dev.42",
+    "@itwin/core-telemetry": "^3.1.0-dev.42",
+    "@itwin/ecschema-metadata": "^3.1.0-dev.42",
     "@itwin/electron-authorization": "^0.8.3",
-    "@itwin/express-server": "^3.0.0",
-    "@itwin/desktop-viewer-react": "2.0.5",
+    "@itwin/express-server": "^3.1.0-dev.42",
+    "@itwin/desktop-viewer-react": "3.0.0-dev.1",

```

Comment:
i _think_ we can set all these viewer dependencies to `"workspace:*"`

Reply:
ah, for some reason i thought we wre already usign worskapces, thought it was baked into rush. rush should be smart enough when handling bumps, but yea we can tackle it later. 

these 2 comments are nits, so feel free to ignore and merge into next whenever you want